### PR TITLE
feat: synchronize with svc-gis

### PR DESCRIPTION
**Pull request by amsmith-pro**
Adds a bunch of waypoints and some no-fly zones to the Noord-Holland area for demo purposes, also updates the zones interaction with svc-gis

### CODE REVIEW

File: .cspell.project-words.txt [hunk 1]
```diff
@@ -40,3 +40,8 @@ adsb
 oneshot
 thiserror
 Refactorings
+schipol
```
Comment (owlot) [inline .cspell.project-words.txt]: missing 'h', might recognize it if you fix the typo in the code

File: server/src/region/nl.rs [hunk 2]
```diff
@@ -74,17 +74,23 @@ impl RegionInterface for super::RegionImpl {
...
+            (4.7091866, 52.3827247),
+            (4.6507947, 52.3294647),
+            (4.7560834, 52.2572307),
+            (4.8234058, 52.3214912),
+            (4.7091866, 52.3827247),
         ];
 
         from_remote.insert(
-            "ARROW-NL-TFR-ZONE".to_string(),
+            "ARROW-NL-NOFLY-SCHIPOL".to_string(),
```
Comment (owlot) [inline server/src/region/nl.rs]: Here's the missing 'H' `ARROW-NL-NOFLY-SCHIPHOL`

Comment (amsmith-pro) [reply server/src/region/nl.rs]: 🤦  https://github.com/Arrow-air/svc-compliance/pull/16/commits/f4d6fe8cd6bd04e5872a2b94a5f6e3b73ababb50

File: .cspell.project-words.txt (hunk 1 above)
Comment (amsmith-pro) [reply .cspell.project-words.txt]: https://github.com/Arrow-air/svc-compliance/pull/16/commits/f4d6fe8cd6bd04e5872a2b94a5f6e3b73ababb50

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+configs:
+  log4rs:
+    file: log4rs.yaml
+  dot-env:
+    file: .env
+
+services:
+  web-server:
+    depends_on:
+      redis:
```
Comment (owlot) [inline docker-compose.yml]: test is failing cause there's no redis service defined in this file but web-server depends on it.
I think the dependency can be removed here

Comment (amsmith-pro) [reply docker-compose.yml]: ahh ok - https://github.com/Arrow-air/svc-compliance/pull/16/commits/243550b9f8c08e4cb0333db52ee6a01fda6ce2dd

**Review by owlot — APPROVED** [review]

**Pull request merged**